src/deno_json/mod.rs

```diff
@@ -1016,24 +1033,27 @@ impl ConfigFile {
 
   /// Returns true if the configuration indicates that JavaScript should be
   /// type checked, otherwise false.
-  pub fn get_check_js(&self) -> bool {
+  pub fn check_js(&self) -> Option<bool> {
```

Comment:
Argh, I hate `Option<bool>`. What does `None` mean in this case? Consider using a 3-state enum instead

Reply:
Now I see that it's used later. Consider updating the docstring instead of adding an enum

Reply:
I think `Option<bool>` is fine. `None` means not set